main_eval.py

```diff
@@ -0,0 +1,103 @@
+import argparse
+import json
+import logging
+import fnmatch
+import wandb
+from pathlib import Path
+from typing import Union
+import yaml
+from pydantic import BaseModel
+
+from lm_eval import tasks, evaluator
+
+logging.getLogger("openai").setLevel(logging.WARNING)
+
+
+def load_config(path: Union[str, Path]):
+    with open(path, "r") as stream:
+        try:
+            return yaml.safe_load(stream)
+        except yaml.YAMLError as exc:
+            print(exc)
+
+
+class EvalPipelineConfig(BaseModel):
```

Comment:
Do you think we'll need several evaluation scripts? If so it might be worth moving the configuration somewhere else so we can reuse it.

Reply:
Thanks, I have made this change now :)